docker/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
       - redis:/data:z
 
   timescale:
-    image: timescale/timescaledb:latest-pg12
+    image: timescale/timescaledb:1.7.4-pg12
```

Comment:
We are not sure about this. Let's wait to hear from ruaok

Reply:
Yup! This is one of the makeshift fixes I needed to make for local dev.